src/hs_config.h

```diff
@@ -53,6 +53,7 @@ typedef struct hs_config
 {
   char* run_path;
   char* load_path;
+  unsigned load_interval;
```

Comment:
move this down with the other unsigned vars there is no reason to introduce extra padding into the struct.